packages/web-client/app/templates/card-pay/wallet.hbs

```diff
@@ -0,0 +1,119 @@
+{{page-title "Wallet"}}
+
+<section class="card-pay-dashboard__wallet" role="tabpanel" aria-labelledby="card-pay.wallet">
+  <CardPay::DashboardPanel @panel={{@model.panel}} @noBottomPadding={{true}} @noCollapse={{or (not this.layer2Network.isConnected) (is-empty this.prepaidCards)}}>
+    <:detail>
+      {{#each @model.panel.sections as |section|}}
+        <CardPay::DashboardPanel::Section @section={{section}} class="dashboard-panel-section-wallet">
+          <:cta>
+            <Boxel::Button
+              {{on "click" (fn this.transitionToWallet section.workflow)}}
+              @kind="primary"
+              @size="touch"
+              @disabled={{section.isCtaDisabled}}
+              data-test-workflow-button={{section.workflow}}
+            >
+              {{section.cta}}
+            </Boxel::Button>
+
+            <small class="dashboard-panel-section__disclaimer">
+              This is possible if you have a balance of {{network-display-info "layer2" "daiToken"}} in your {{network-display-info "layer2" "fullName"}} wallet.
+              (You can <LinkTo @route="card-pay.deposit-withdrawal">deposit DAI</LinkTo>
+              from your {{network-display-info "layer1" "conversationalName"}} wallet to get a balance of {{network-display-info "layer2" "daiToken"}} in your {{network-display-info "layer2" "fullName"}} wallet.)
+            </small>
+
+            <div class="dashboard-panel-section-wallet__cards">
+              <img src="/images/dashboard/cards@3x.webp" alt="Stack of Cardstack prepaid cards">
+            </div>
+          </:cta>
+        </CardPay::DashboardPanel::Section>
+      {{/each}}
+
+      <section class="dashboard-panel-section">
+        <div class="dashboard-panel-section__body">
+          <p class="dashboard-panel-section__desc">
+            {{svg-jar "card-wallet-app-icon" alt="" role="presentation" height="58"}}
+            <h3 class="dashboard-panel-section__title">Buy a Prepaid Card</h3>
+            <p class="dashboard-panel-section__desc">With Card Wallet you can buy a Prepaid Card directly in the mobile application.</p>
+            <div class="dashboard-panel-section__badges">
+              <div>
+                <a href={{this.testFlightLink}} target="_blank" rel="noopener noreferrer"><img src="/images/icons/download-on-the-app-store-badge.png" alt="Download on the App Store"></a>
+              </div>
+              <div>
+                <a href="#" onclick="return false"><img src="/images/icons/google-play-badge.png" class="dashboard-panel-section__badges-google-play-badge" alt="Get it on Google Play"></a>
```

Comment:
Should we omit the href and onclick here, and add `cursor: not-allowed`?

Reply:
Sounds like a good idea!

Reply:
Or, we can just remove the link for now and only display the image